apps/console/src/features/groups/components/wizard/group-basics.tsx

```diff
@@ -197,93 +206,108 @@ export const GroupBasics: FunctionComponent<GroupBasicProps> = (props: GroupBasi
-                                            message: t("console:manage.features.groups.notifications.fetchGroups." +
-                                                "genericError.message")
-                                        }));
-                                    });
-                                }
-                            } }
-                            value={ initialValues && initialValues.groupName }
-                            loading={ isRegExLoading }
-                        />
-                    </GridColumn>
-                </GridRow>
-            </Grid>
-        </Forms>
+
+                                        const searchData: SearchGroupInterface = {
+                                            filter: `displayName eq  ${ userStore }/${ value }`,
+                                            schemas: [
+                                                "urn:ietf:params:scim:api:messages:2.0:SearchRequest"
+                                            ],
+                                            startIndex: 1
+                                        };
+
+                                        await searchGroupList(searchData).then(response => {
+                                            if (response?.data?.totalResults !== 0) {
+                                                validation.isValid = false;
+                                                validation.errorMessages.push(
+                                                    t("console:manage.features.roles.addRoleWizard." +
+                                                        "forms.roleBasicDetails.roleName.validations.duplicate",
+                                                        { type: "Group" }));
+                                            }
+                                        }).catch(() => {
+                                            dispatch(addAlert({
+                                                description: t("console:manage.features.groups.notifications." +
+                                                    "fetchGroups.genericError.description"),
+                                                level: AlertLevels.ERROR,
+                                                message: t("console:manage.features.groups.notifications.fetchGroups." +
+                                                    "genericError.message")
+                                            }));
+                                        });
+                                    }
+                                } }
+                                value={ initialValues && initialValues.basicDetails?.groupName }
+                                loading={ isRegExLoading }
+                            />
+                        </GridColumn>
+                    </GridRow>
+                </Grid>
+            </Forms>
+            <h5>{ t("console:manage.features.roles.addRoleWizard.wizardSteps.2") }</h5>
```

Comment:
Better to use `Header` from semantic or `Heading` from `@wso2is/react-component` here.

Reply:
Fixed by 60c943b